### ISSUE TITLE
script-src: 'unsafe-inline' content security policy fails with pdfmake

### ISSUE THREAD

**Issue by bmclain** [bpampuch/pdfmake#1371]
I'm sorry if I'm being too lazy to not grep through pdfmake to find the relevant line of code, but I ran into a CSP issue while using the DataTables library that bundles pdfmake as a plugin. I'm using the following CSP:

`script-src: 'unsafe-inline'`

It throws the following error when using pdfmake via DataTables (they bundle it as a plugin), version 0.1.32:

```Uncaught EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src https: 'unsafe-inline'".

    at new Function (<anonymous>)
    at new Pointer (datatables.js:66911)
    at Object.<anonymous> (datatables.js:48358)
    at Object.<anonymous> (datatables.js:62782)
    at __webpack_require__ (datatables.js:9199)
    at Object.<anonymous> (datatables.js:47785)
    at Object.<anonymous> (datatables.js:47855)
    at Object.<anonymous> (datatables.js:47857)
    at __webpack_require__ (datatables.js:9199)
    at Object.<anonymous> (datatables.js:47708)
```

Here is the relevant piece of code:

```/* 121 */
/***/ (function(module, exports, __webpack_require__) {

	// Generated by CoffeeScript 1.7.1
	(function() {
	  var Pointer, VoidPointer, utils;

	  utils = __webpack_require__(110);

	  Pointer = (function() {
	    function Pointer(offsetType, type, options) {
	      var _base, _base1, _base2, _base3;
	      this.offsetType = offsetType;
	      this.type = type;
	      this.options = options != null ? options : {};
	      if (this.type === 'void') {
	        this.type = null;
	      }
	      if ((_base = this.options).type == null) {
	        _base.type = 'local';
	      }
	      if ((_base1 = this.options).allowNull == null) {
	        _base1.allowNull = true;
	      }
	      if ((_base2 = this.options).nullValue == null) {
	        _base2.nullValue = 0;
	      }
	      if ((_base3 = this.options).lazy == null) {
	        _base3.lazy = false;
	      }
	      if (this.options.relativeTo) {
	        this.relativeToGetter = new Function('ctx', "return ctx." + this.options.relativeTo); // CSP ERROR HERE
	      }
	    }
```

**Comment by liborm85** [comment]
This is in the third party library, see https://github.com/bpampuch/pdfmake/issues/1360.

**Comment by liborm85** [comment]
Issue fixed in version 0.2.0 - https://github.com/bpampuch/pdfmake/issues/2282.
